src/testCombiner.ts

```diff
@@ -120,12 +121,21 @@ export function genTestClass(
     annotation = shortenClassNames({ imports: testImports, body: annotation });
   }
 
-  let rule = '@org.junit.Rule';
-  if (testImports && testImports.length) {
-    if (testImports.indexOf('org.junit.Rule') === -1) {
-      testImports.push('org.junit.Rule');
-    }
-    rule = '@Rule';
+  if (classRules.size > 0 && (testImports.indexOf('org.junit.Rule') === -1)) {
+    testImports.push('org.junit.Rule');
+  }
+
+  let rulesString = '';
+
+
+  for (const s of combinedTests.classRules) {
+    rulesString = rulesString + ' ' + s.replace(/^(@[\w.]+)\s+/, ' $1\n  ') + '\n\n';
+  }
+
+  rulesString = rulesString.slice(0, -2);
+  rulesString = shortenClassNames({ imports: testImports, body: rulesString });
```

Comment:
You could probably then do something like this
```suggestion
  let formattedRules = shortenClassNames({ imports: testImports, body: mappedRules });
```